ludwig/schema/decoders/base.py

```diff
@@ -85,7 +85,7 @@ class BaseDecoderConfig(schema_utils.BaseMarshmallowConfig, ABC):
 
 
 @DeveloperAPI
-@register_decoder_config("passthrough", [BINARY, CATEGORY, NUMBER, SET, VECTOR, SEQUENCE, TEXT])
+@register_decoder_config("passthrough", [BINARY, CATEGORY, SET, VECTOR, SEQUENCE, TEXT])
```

Comment:
@arnavgarg1 I thought we had disabled the passthrough decoder?

Reply:
I think we removed it from the encoder registry, but not from the schema.